Product.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using InventoryManagementSystem.LoggableInterface;
+
+namespace InventoryManagementSystem.ProductInfo
+{
+    public class Product : ILoggable
+    {
+        public int ProductQuantity { get; set; }
+        public string ProductName { get; set; }
+        public double? ProductPrice { get; set; }
```

Comment:
It might be enough here to have only the variable name as its purpose: Quantity, Name, Price; because when you use them it will give duplicate names. For example in the inventory file: `product.ProductName = newName` "product" is written twice.